dashboard/src/components/Common/Autocomplete/AutocompleteField.jsx

```diff
@@ -36,7 +37,9 @@ class AutocompleteField extends React.Component {
 
   renderSuggestion(suggestion) {
     return (
-      <span>{suggestion.alias}</span>
+      <div onMouseOver={this.handleHover.bind(this, suggestion.alias)}>
```

Comment:
I think you're right on the second suggestion, so I updated for that route. No reason to have a separate handleHover in this case.